src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
```

Comment:
The interface name 'Area' is a noun, which is not recommended for interface names. Interface names should be descriptive and often include adjectives or nouns that describe a capability, such as 'Calculable' or in this case, 'AreaCalculable' could be more appropriate.